packages/next/lib/format-server-error.ts

```diff
@@ -0,0 +1,12 @@
+export function formatServerError(error: Error): void {
+  if (error.message.includes('createContext is not a function')) {
+    const message =
+      'createContext only works in Client Components. Add the "use client" directive at the top of the file to use it. Read more: https://beta.nextjs.org/docs/rendering/server-and-client-components#context'
```

Comment:
This should use an error link instead so that it can be updated without a new release of Next.js 👍

Reply:
Thanks!